### DISCUSSION TITLE
Can reflex run on Windows 10 and Windows 11?

### DISCUSSION THREAD

**Discussion by usuall** [reflex-dev/reflex#2199] [Q&A]
Can reflex run on Windows 10 and Windows 11?
I have to work on entering the data of the website through selenium on the local PC.

Can I create it as a single file with a library such as pyinstaller or cx_Freeze or nuitka?

I'm going to create an app that runs locally.
If you have a tutorial that you make as a solo file, please introduce it.

**Comment by timmypidashev** [comment]
The recommended way to run a reflex app on Windows is using python's venv(virtual environment). Since you seem to be looking for a more unique solution however, I would recommend taking a look at [Windows Subsystem For Linux(WSL)](https://learn.microsoft.com/en-us/windows/wsl/). Using WSL you can pretty much abstract away all the edge cases Windows may have. Another good solution would be to dockerize your application. There is a great example in the reflex repository [here](https://github.com/reflex-dev/reflex/tree/main/docker-example).

There is also some good documentation in reflex's documentation regarding installation [here](https://github.com/reflex-dev/reflex/tree/main/docker-example).
